mighty/mighty_agents/base_agent.py

```diff
@@ -431,7 +431,7 @@ def update(self, metrics: Dict, update_kwargs: Dict) -> Dict:
 
             metrics.update(agent_update_metrics)
 
-            metrics = {k: np.array(v) for k, v in metrics.items()}
+            # metrics = {k: np.array(v) for k, v in metrics.items()}
```

Comment:
Not sure why this is commented out

Reply:
Ah, this doesn't belong on this branch, true. It's commented out because it causes bugs, so I deactivated it locally (see #60 ). @amsks How do we deal with this? I can comment it back in or remove it entirely. I think the functionality of this line is 10-ish lines down.

Reply:
AS discussed, this is resolved